docs/commandline.md

```diff
@@ -18,9 +18,11 @@ If you use Maven you can have a look [here](https://maven.apache.org/plugins/mav
 You can use the command `gear` under the bin directory to submit, query and terminate an application:
 
 ```bash
-gear app [-namePrefix <application name prefix>] [-conf <custom gearpump config file>] -jar xx.jar MainClass <arg1> <arg2> ...
+gear app [-namePrefix <application name prefix>] [-executors <number of executors to launch>] [-conf <custom gearpump config file>] -jar xx.jar MainClass <arg1> <arg2> ...
 ```
 
+If you not specify the option ```executors```, the number of executors to be launched will be the same as living number of Workers.
```

Comment:
Maybe we should just use 1 as the default value. Maybe it is safer compared with launching as mang executors as workers.